doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -424,6 +426,8 @@ Bluetooth libraries and services
     * The :kconfig:option:`BT_MESH_MODEL_SRV_STORE_TIMEOUT` Kconfig option, that is controlling timeout for storing of model states, is replaced by the :kconfig:option:`BT_MESH_STORE_TIMEOUT` Kconfig option.
 
   * Fixed an issue where the :ref:'bt_mesh_dtt_srv_readme' model could not be found for models spanning multiple elements.
+  * Fixed an issue where the :ref:'bt_mesh_sensor_srv_readme' model could add a corrupted marshalled sensor data into the Sensor Status message due to the fetched sensor value being outside of range.
```

Comment:
```suggestion
  * Fixed an issue where the :ref:'bt_mesh_sensor_srv_readme' model would add a corrupted marshalled sensor data into the Sensor Status message because the fetched sensor value was outside of range.
```